srml/system/src/lib.rs

```diff
@@ -218,24 +219,35 @@ decl_module! {
 			Self::deposit_event_indexed(&[], event);
 		}
 
+		/// A big dispatch that will disallow any other transaction to be included.
+		// TODO: this must be preferable available for testing really (not possible at the moment).
+		#[weight = SimpleDispatchInfo::MaxOperational]
+		fn fill_block(origin) {
```

Comment:
I merely added this for a test (which is not that important actually). It should probably be removed unless if someone can find another use case for it OR claim that as a root function it is safe to exist?

Reply:
can't we feature-gate it on tests?

Reply:
I never tried it but I recall that apparently, we don't allow most `#[cfg]`s in `decl_module!`. As the comment says: 

> // TODO: this must be preferable available for testing really (__not possible at the moment__).


will try.

Reply:
Looks like: no, we can't.

Reply:
Can you not use the `test-runtime`?